### ISSUE TITLE
Compile error: unable to build stream

### ISSUE THREAD

**Issue by dgroup** [Lec1k/task-manager#6]
## Input

``` cmd
   git clone https://github.com/Lec1k/task-manager.git
   cd task-manager
   mvn clean install
```
## Output

``` java
"C:\Program Files\Java\jdk1.8.0_20\bin\java" -Xmx512m -XX:MaxPermSize=256m -Dmaven.home=D:\Netcracker\_soft\apache-maven-3.0.4-nc -Dclassworlds.conf=D:\Netcracker\_soft\apache-maven-3.0.4-nc\bin\m2.conf -Didea.launcher.port=7532 "-Didea.launcher.bin.path=C:\Program Files (x86)\JetBrains\IntelliJ IDEA 12.1.3\bin" -Dfile.encoding=UTF-8 -classpath "D:\Netcracker\_soft\apache-maven-3.0.4-nc\boot\plexus-classworlds-2.4.jar;C:\Program Files (x86)\JetBrains\IntelliJ IDEA 12.1.3\lib\idea_rt.jar" com.intellij.rt.execution.application.AppMain org.codehaus.classworlds.Launcher --fail-fast --strict-checksums clean install
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building task-manager 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.4.1:clean (default-clean) @ task-manager ---
[INFO] Deleting D:\Netcracker\JavaCourse\AlexeyVoronenko\task-manager\target
[INFO] 
[INFO] --- maven-resources-plugin:2.5:resources (default-resources) @ task-manager ---
[debug] execute contextualize
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 4 resources
[INFO] 
[INFO] --- maven-compiler-plugin:2.3.2:compile (default-compile) @ task-manager ---
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 12 source files to D:\Netcracker\JavaCourse\AlexeyVoronenko\task-manager\target\classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[7,27] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,20] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,38] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,49] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,16] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,34] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,45] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,40] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,58] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,69] error: package Alert does not exist
[INFO] 19 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.684s
[INFO] Finished at: Wed Jan 06 13:43:23 EET 2016
[INFO] Final Memory: 11M/155M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project task-manager: Compilation failure: Compilation failure:
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[7,27] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,20] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,38] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[92,49] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[102,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,16] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,34] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[127,45] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\Controller.java:[137,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,12] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,30] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Controller\addDialogueController.java:[72,41] error: package Alert does not exist
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,40] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,58] error: cannot find symbol
[ERROR] \Netcracker\JavaCourse\AlexeyVoronenko\task-manager\src\main\java\app\Model\Notifier.java:[32,69] error: package Alert does not exist
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0

Process finished with exit code 1
```

**Comment by alexey-voronenko** [comment]
Did the same steps 
git clone https://github.com/Lec1k/task-manager.git
   cd task-manager
   mvn clean install
build success
![image](https://cloud.githubusercontent.com/assets/16180125/12174614/51e29402-b566-11e5-991b-987d686f55c4.png)

**Comment by dgroup** [comment]
It depends of JVM. The jfx is available since 1.8u60.

My version was 1.8u20 therefore i got compile error.
